packages/expo-localization/plugin/src/withExpoLocalization.ts

```diff
@@ -0,0 +1,32 @@
+import {
+  AndroidConfig,
+  ConfigPlugin,
+  createRunOncePlugin,
+  withStringsXml,
+} from 'expo/config-plugins';
+
+const pkg = require('expo-localization/package.json');
+
+const withExpoLocalization: ConfigPlugin = (config) => {
+  if (config.extra?.supportsRTL === undefined) return config;
```

Comment:
Try to avoid explicit references to `undefined` and using triple equals on both `null` and `undefined`. Since `null !== undefined`, this could lead to errors when someone explicitly passes `null` instead of leaving this field unset. Use `== null` instead

```suggestion
  if (config.extra?.supportsRTL == null) return config;
```